uberfire-extensions/uberfire-wires/uberfire-wires-bpmn/uberfire-wires-bpmn-api/src/main/java/org/uberfire/ext/wires/bpmn/api/type/BpmnResourceTypeDefinition.java

```diff
@@ -16,10 +16,27 @@
 package org.uberfire.ext.wires.bpmn.api.type;
 
 import org.uberfire.backend.vfs.Path;
+import org.uberfire.ext.wires.bpmn.api.category.Process;
+import org.uberfire.workbench.category.Category;
 import org.uberfire.workbench.type.ResourceTypeDefinition;
 
 public class BpmnResourceTypeDefinition implements ResourceTypeDefinition {
 
+    private Category category;
+
+    public BpmnResourceTypeDefinition() {
+
+    }
+
+    public BpmnResourceTypeDefinition(final Process category) {
```

Comment:
This works?

Reply:
The important there is the subclass that has the injections and it is the only class that uses it.